.travis.yml

```diff
@@ -89,30 +87,49 @@ before_script:
 - ./tools/setup.py
 script:
 - |-
-  set -e
+  # Check lint and format.
+  set -e # Fail immediately if any of the following fail.
   ./tools/lint.py
   ./tools/test_format.py
+
+- |-
+  # LSAN build of test_cc. We are in the process of getting a
+  # completely clean LSAN build, but it will take some work. So for now
+  # we just run a single test that passes under LSAN.
+  echo is_asan=true >> target/debug/args.gn
```

Comment:
A little hacky ... DENO_BUILD_ARGS breaks down a little bit here. 
I'm wondering if it'd be better to add a separate 'lsan' config.

Reply:
DENO_BUILD_ARGS applies the release build too.. so it can't be used here. And I think adding another env var for this (EG DENO_DEBUG_BUILD_ARGS) is more machinery than its worth